src/model/block/ISelectOneResponseBlockConfig.ts

```diff
@@ -20,9 +20,35 @@
 import {IBlockConfig} from '../..'
 
 export interface ISelectOneResponseBlockConfig extends IBlockConfig {
-  prompt: string
-  question_prompt?: string
-  choices: StringMapType
+  prompt?: Uuid
+  question_prompt?: Uuid
+  choices: Record<string, Choice>
+  ivr?: IvrConfig
 }
 
-type StringMapType = {[k: string]: string}
+interface Choice {
+  name: string
+  ivr_test?: IvrTest
+  text_tests?: TextTest[]
+  prompt: Uuid
+}
+
+interface IvrTest {
+  /** @format floip-expression */
+  test_expression: string
+}
+
+interface TextTest {
+  language?: string
+
+  /** @format floip-expression */
+  test_expression: string
+}
+
+interface IvrConfig {
+  digit_prompts?: Uuid[]
```

Comment:
Can we get confirmation from @seifertk on this ? if this represents what we meant by `array of resources`